packages/openapi-typescript/src/index.ts

```diff
@@ -184,43 +185,15 @@ async function openapiTS(schema: string | URL | OpenAPI3 | Readable, options: Op
           break;
         }
         case "RequestBodyObject": {
-          subschemaOutput = transformRequestBodyObject(subschema.schema, { path, ctx: { ...ctx, indentLv } });
+          subschemaOutput = `${transformRequestBodyObject(subschema.schema, { path, ctx: { ...ctx, indentLv } })};`;
           break;
         }
         case "ResponseObject": {
-          subschemaOutput = transformResponseObject(subschema.schema, { path, ctx: { ...ctx, indentLv } });
+          subschemaOutput = `${transformResponseObject(subschema.schema, { path, ctx: { ...ctx, indentLv } })};`;
           break;
         }
         case "SchemaMap": {
-          subschemaOutput += "{\n";
-          indentLv++;
-
-          outer: for (const [name, schemaObject] of getEntries(subschema.schema!)) {
```

Comment:
Clean up a lot of duplicated transforming of a map of schema objects with `transformSchemaObjectMap()`.